config/initializers/delayed_job.rb

```diff
@@ -1,2 +1,21 @@
 Delayed::Worker.destroy_failed_jobs = false
 Delayed::Worker.max_attempts = 1
+
+module Delayed
```

Comment:
Trivial -- why put this in `Delayed::Plugins` when we can instead make our own class in like `lib` or `services` called `ExtraJobAttributes`, then do `Delayed::Worker.plugins << ExtraJobAttributes`? I'd prefer the class definition to be outside the initializer.

Not a blocker.